frontend/src/utils/MadLibs.ts

```diff
@@ -229,7 +231,12 @@ const CATEGORIES_LIST: Category[] = [
     title: 'Community Safety',
     definition: '',
     options: COMMUNITY_SAFETY_DROPDOWNIDS as unknown as DropdownVarId[],
-  }
+  },
+  {
+    title: 'Maternal Health',
+    definition: '',
+    options: MATERNAL_HEALTH_CATEGORY_DROPDOWNIDS as unknown as DropdownVarId[],
+  },
```

Comment:
sorry it isn't scoped out, but can we put this behind an env flag? you can follow the same pattern as `VITE_SHOW_GUN_VIOLENCE`

Reply:
https://healthequitytracker.notion.site/Merging-behind-a-feature-flag-4e12dc6aac7e496b870cd7d895df7ea1?pvs=4

Reply:
@JDemlow great, a few more steps needed for the feature flag:

- [ ] set that variable equal to true in the `.env.deploy_preview` and `.env.staging` files (and your local `.env.development if you want to see it locally; that env isn't checked in to git)
- [ ] now, the line of code you wrote will import the true/false state of the flag based on the environment it's in (netlify deploy, staging = dev site, or prod where it should still be false)
- [ ] we also want to conditionally show/hide that new dropdown MadLib option based on the flag. So for the code above, instead of directly declaring the maternal health object in the array, you'd want to conditionally push it onto the array. Something like: 

```
if (SHOW_NEW_MATERNAL_MORTALITY) CATEGORIES_LIST.append(
  {
    title: 'Maternal Health',
    definition: '',
    options: MATERNAL_HEALTH_CATEGORY_DROPDOWNIDS as unknown as DropdownVarId[],
   }
)
```